Analytics-CSharp/Segment/Analytics/Types.cs

```diff
@@ -27,16 +27,7 @@ internal void ApplyRawEventData(RawEvent rawEvent)
             this.timestamp = rawEvent.timestamp;
             this.context = rawEvent.context;
             this.integrations = rawEvent.integrations;
-        }
-
-        internal async Task ApplyRawEventData(Store store)
-        {
-            var userInfo = await store.CurrentState<UserInfo>();
-            if (userInfo.isNull) return;
-
-            this.anonymousId = userInfo.anonymousId;
-            this.userId = userInfo.userId;
-            this.integrations = new JsonObject();  
+            this.integrations = new JsonObject();
```

Comment:
this shouldn't change

Reply:
I was trying to reproduce the change I had made on my first attempt at this, and clearly forgot the point.  Fixing...